README.adoc

```diff
@@ -181,11 +204,24 @@ Navigate to `{name-api}` and you will see that the greeting message has changed
 
 == Testing the microservices
 
-Run the tests by running the following command and substituting `[hostname]` for the correct value.
+****
+[system]#*{win} | {mac}*#
+
+Run the integration tests against a cluster running with a hostname of localhost:
+
+```
+mvn verify -Ddockerfile.skip=true -Dcluster.ip=localhost
+```
+
+[system]#*{linux}*#
+
+Run the integration tests against a cluster running at Minikube's IP address:
 
 ```
-mvn verify -Ddockerfile.skip=true -Dcluster.ip=[hostname]
+mvn verify -Ddockerfile.skip=true -Dcluster.ip=`minikube ip`
```

Comment:
Let us present `minikube ip` consistent with how we did it in the earlier section?  We used [hostname] there too?

Or, we go for the command that doesn't need cluster.ip?

Reply:
@yeekangc `minikube ip` is actually a command. If you run this as is (ie. copy and paste it), it will automatically substitute `minikube ip` for the actual ip.

Reply:
Great.  It is hard to tell what we are trying to say from the PR.